tests/simple-usage-spec.js

```diff
@@ -26,6 +26,33 @@ describe('rule engine usage', function() {
       });
   });
 
+  it('should match a simple regexp rule', function(done) {
+    rEngine.apply([{'view': { val: 'abcd' }}], [{'view': {val: /abcd/, should: true}}])
+      .then(function(res){
+        expect(res).toBe(true);
```

Comment:
That's right! I did it this way to ease the cleanup when #2 will be done, so we can remove `.then(done).catch(done)` in a breeze